cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -418,16 +414,9 @@ private void createConfigTable() throws IOException {
   }
 
   @Override
-  public void dropAll() throws Exception {
-    dropTablesWithPrefix(tableNamePrefix);
-    // drop config table last
-    drop(Bytes.toBytes(configTableName));
-  }
-
-  @Override
   public void dropAllInNamespace(String namespaceId) throws Exception {
     // Note: The trailing "." is crucial, since otherwise nsId could match nsId1, nsIdx etc
-    dropTablesWithPrefix(String.format("%s.%s.", tableNamePrefix, namespaceId));
+    dropTablesWithPrefix(String.format("%s.", getTableNamePrefix(namespaceId)));
```

Comment:
slightly skeptical about this approach. If we depend on prefix search, how will we scan all tables in a namespace with hbases? We'll probably have to implement an API - ``tableUtil.listTablesInNamespace(namespaceId, prefix)``. Nothing you can do in this PR though (unless you have better approaches in mind)